app/src/main/java/hacking/to/the/gate/CircleCollider.java

```diff
@@ -0,0 +1,23 @@
+package hacking.to.the.gate;
+
+/**
+ * Created by Ruiqian on 8/31/2015.
+ */
```

Comment:
I don't quite feel it's a good idea to let jet, powerup and bullets to extends this class. When there is no obvious reason, you should prefer composition than inheritance. And in this case, the parent class is just used as a data object.